### PR TITLE
In the mobile version, the screen area shifts to the right because of the iframe video

### PR DESCRIPTION
**Pull request by thinkjazz**
### How it was fixed
1. Into TheParse.ts into 223 line was added `return <div class="video-container"><iframe width="500" height="282" src="${encodeURI(embed)}" allowfullscreen class="youtube" frameborder="0"></iframe></div>;`
2. Added a style patch with rules for div="video-container", which is taken from the database, which was put there by the parser **(!MPORTANT This will only work for new video embeds from YouTubez For older posts that do not have a video class in the iframe and do not have a wrapper `<div="video-container"><iframe .......> </iframe></div>`  )** 
_So, there will be a need for migration! I had a weird side effect with the video stretching, but I couldn't reproduce it._
3. So far I haven't come up with anything smarter than importing styles into the PostComponent.tsx component
video-convert-1660006128000
full_hd
iphone
iphone_h
pixel

### CODE REVIEW

File: frontend/src/Components/PostContentVideo.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  height: 0;
+  padding-bottom: 56.25%;
+}
+.youtube {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
```
Comment (pazoozoo42) [inline frontend/src/Components/PostContentVideo.css:13]: minior code style: I'd separate rules and add a new line at the end of the file to get rid of Github warning:
```css
.video-container {
  position: relative;
  width: 100%;
  height: 0;
  padding-bottom: 56.25%;
}

.youtube {
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}
```

**Review by pazoozoo42 — COMMENTED** [review]
Looks good and definitely an important improvement for mobile users!

My only concern is that 100% would make videos way too big on large screens, take a look: 

image

Maybe there's a way to apply max-width for it and keep it responsive (so that it could get smaller) for small screens?

File: frontend/src/Components/PostContentVideo.css (hunk 1 above)
Comment (thinkjazz) [reply frontend/src/Components/PostContentVideo.css]: 1)100% yes maybe it's make sense!  for big screens make a less width video frame. But reducing the width`(width:90%) `rule entails that the block with the video and the block with the `textarea  `will be unequal and unbalanced.  And I would like to video equal to the size of the field text area) 
2) Done
Screenshot 2022-08-11 102527
Screenshot 2022-08-11 102601

File: backend/src/parser/TheParser.ts [hunk 2]
```diff
@@ -221,6 +221,7 @@ export default class TheParser {
 
         // noinspection HtmlDeprecatedAttribute
         return `<iframe width="500" height="282" src="${encodeURI(embed)}" allowfullscreen frameborder="0"></iframe>`;
+
```
Comment (pazoozoo42) [inline backend/src/parser/TheParser.ts:224]: let's remove this newline so that this file stays untouched

**Review by pazoozoo42 — APPROVED** [review]
looks good to me, thanks

File: backend/src/parser/TheParser.ts (hunk 2 above)
Comment (thinkjazz) [reply backend/src/parser/TheParser.ts]: done

**Review by Aivean — APPROVED** [review]

**Review by pazoozoo42 — APPROVED** [review]

**Pull request merged**